aissemble-open-inference-protocol-shared/src/aissemble_open_inference_protocol_shared/config/oip_config.py

```diff
@@ -19,11 +19,37 @@ class OIPConfig:
     DEFAULT_ALGORITHM = "HS256"
     DEFAULT_PDP_URL = "http://localhost:8080/pdp"
 
+    DEFAULT_GRPC_HOST = "0.0.0.0"
+    DEFAULT_GRPC_PORT = "8081"
+    DEFAULT_GRPC_WORKERS = "3"
+    DEFAULT_AUTH_ENABLED = "false"
+
+    DEFAULT_FASTAPI_HOST = "127.0.0.1"
+    DEFAULT_FASTAPI_PORT = "8082"
+    DEFAULT_FASTAPI_RELOAD = "True"
+
     def __init__(self):
         self.properties = PropertyManager.get_instance().get_properties(
             "oip.properties"
         )
 
+    #########################
+    # Authorization
+    #########################
+
+    @property
+    def grpc_auth_enabled(self) -> bool:
```

Comment:
Q/A: Is there a use case to differentiate by technology instead of
```suggestion
    def auth_enabled(self) -> bool:
```

Reply:
Just talked with the team and we agreed we should keep this consistent across impls. Going to set it as `auth_enabled` default `true` and have fastapi also leverage it